docker-images/kafka/Makefile

```
@@ -8,6 +8,7 @@ clean:
 .kafka-agent.tmp: ../../kafka-agent/target/kafka-agent-$(RELEASE_VERSION).jar
 	test -d tmp || mkdir tmp
 	cp ../../kafka-agent/target/kafka-agent-$(RELEASE_VERSION).jar tmp/
+	sh "mvn dependency:copy-dependencies -f kafka-thirdparty-libs"
```

As mention in the comment, I think we need to add `../../`

```suggestion
	sh "mvn dependency:copy-dependencies -f ../../kafka-thirdparty-libs"
```

---

I will add, but i do not really understand why.
As I understand context path for `mvn` command is `docker-images/kafka/` 
But i ll never use make file previously, so may be i miss something.

---

@scholzj No problem, I moved because you say that threre is know reason to keep it as `maven` submodule. So i decide to keep it in folder with docker images details, as it use only for docker images.